src/helpers/string.ts

```diff
@@ -0,0 +1,3 @@
+export const camelize = (str: string): string => {
+  return str.toLowerCase().replace(/[^a-zA-Z0-9]+(.)/g, (m, chr) => chr.toUpperCase());
+};
```

Comment:
Why don't you use the lodash's `camelCase` 🤔 Any difference?

Reply:
ah, there is no difference here. Will remove this file